ui/page/tickets/utils.go

```diff
@@ -457,6 +504,26 @@ func createClickGestures(count int) []*gesture.Click {
 	return gestures
 }
 
+func ticketAgeTimeFormat(secs int) string {
```

Comment:
This method is an exact replica of `nextTicketRemaining` Line 527. Only difference is that `nextTicketRemaining` takes into account weeks.

Reply:
Actually, this method returns the time in a single rounded off unit("1d", "1h", or "5m") but nextTicketRemaining returns the complete time remaining ("1h20m5s").